src/libraries/System.Memory.Data/src/System/BinaryData.cs

```diff
@@ -4,6 +4,7 @@
 using System.ComponentModel;
 using System.Diagnostics.CodeAnalysis;
 using System.IO;
+using System.Net.Mime;
```

Comment:
This is only needed for ref doc comments.

Reply:
The using is fine.